react-props/my-app/src/App.tsx

```diff
@@ -0,0 +1,24 @@
+import { Header } from './Header';
+import { ImageContainer } from './ImageContainer';
+import { ImageCaption } from './ImageCaption';
+import { ImageDescription } from './ImageDescription';
+import { ButtonContainer } from './ButtonContainer';
+import './App.css';
+
+const headerText = 'React Props Text';
+const newSrc = '/vite.svg';
+const newImgCap = 'This is a newer Image Caption :)';
+const newImgDscript = 'This the images new description :) Lets GO!!!!!!!!!';
+const newButton = 'Time for New Image';
+
+export default function App() {
+  return (
+    <>
+      <Header headerText={headerText} />
+      <ImageContainer newSrc={newSrc} />
```

Comment:
The name `newSrc` doesn't make sense to me. It's not new, it's just the src! Why not just use `src`?

Similarly, `imageCaptionText` seems redundant since we know from the component name that it is an ImageCaption. So why not just `text`? And so on.